spark/v3.4/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java

```diff
@@ -165,6 +167,14 @@ public void testCreateTable() {
     Assert.assertEquals("Should be partitioned on 3 columns", 3, third.spec().fields().size());
   }
 
+  @Test
+  public void showView() {
+    sql("DROP VIEW IF EXISTS %s", "test");
+    sql("CREATE VIEW %s AS SELECT 1 AS id", "test");
+    Object[] expected = row("default", "test", false);
+    assertThat(sql("SHOW VIEWS")).contains(expected);
```

Comment:
`expected` can be inlined here:
```suggestion
    assertThat(sql("SHOW VIEWS")).contains(row("default", "test", false));
```

Reply:
Fixed. Thanks for the suggestion!